source/contracts/Augur.sol

```diff
@@ -34,6 +35,33 @@ contract Augur is Controlled, Extractable {
     event TokensBurned(address indexed universe, address indexed token, address indexed target, uint256 amount);
     event FeeWindowCreated(address indexed universe, address feeWindow, uint256 startTime, uint256 endTime, uint256 id);
 
+    mapping(address => bool) private universes;
+
+    //
+    // Universe
+    //
+
+    function createGenesisUniverse() public returns (IUniverse) {
+        return createUniverse(IUniverse(0), bytes32(0));
+    }
+
+    function createChildUniverse(bytes32 _parentPayoutDistributionHash) public returns (IUniverse) {
+        require(universes[msg.sender]);
```

Comment:
Recommend changing this to `require(isValidUniverse(msg.sender))` to make it a tad more clear what the check is doing.